app/utils/cma-hearing-details.ts

```diff
@@ -42,8 +43,32 @@ function getHearingCentre(req: Request): string {
   }
 }
 
+function getHearingCentreEmail(req: Request): string {
+  const hearingCentre = (req.session.appeal.hearing.hearingCentre !== '') ? req.session.appeal.hearing.hearingCentre : req.session.appeal.hearingCentre;
```

Comment:
not sure to understand this line? do we have hearingCentre in two diff places in the session?

Reply:
In short, yes. See @jkumar-moj comment on previous branch regarding a case that is listed and then ended.

It is currently not possible to advance a case using AIP, there will be a ticket to address this soon.

https://github.com/hmcts/ia-aip-frontend/pull/474#discussion_r649011357